packages/cli-platform-apple/src/commands/runCommand/getPropertyFromBuildSettings.ts

```diff
@@ -0,0 +1,41 @@
+import {logger} from '@react-native-community/cli-tools';
+import chalk from 'chalk';
+
+export function getPropertyFromBuildSettings(
```

Comment:
How about getBuildSettings? Currently you need to do all of this parsing and mapping multiple times and you could retrieve `settings[targetIndex].buildSettings` once and then get the data you need from it instead of calling the function multiple times